src/object/ambient_light.cpp

```diff
@@ -46,7 +46,7 @@ AmbientLight::AmbientLight(const ReaderMapping& mapping) :
   if (mapping.get("color", color_vec))
   {
     if (color_vec.size() < 3) {
-      log_warning << "ambient-light requires three float arguments" << std::endl;
+      log_warning << "Ambient-light requires three float arguments" << std::endl;
```

Comment:
Either leave it as `"ambient-light"` (the class name for the object, in its own quotation marks), or change to "Ambient Light".